File: cfgs/pythoncfgs/CHECK_STRING_FOLLOWS_ANBN_PATTERN_NOT.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;conditional;fun_1(foo_1);9;
3;conditional;foo_1 * 2 != foo_2;12;
4;conditional;foo < foo;14;
5;statement;return true;17;
6;conditional;foo_1[foo_2] != 'b';15;
7;statement;return false;16;
8;statement;return false;13;
9;conditional;foo_1[foo_2] != 'a';10;
10;exit;;
Edges
0;2;
1;3;9;
2;4;8;
3;5;6;
4;10;
5;4;7;
6;10;
7;10;
8;2;3;
9;

File: cfgs/java_cfgs_foo/CHECK_STRING_FOLLOWS_ANBN_PATTERN_NOT.java
Nodes
1;special;;;
2;statement;int l = fun_1();14;
3;conditional;(foo_1 % 2) == 1;15;
4;statement;return false;16;
5;exit;;;
6;statement;int i = 0;18;
7;statement;int j = foo_1 - 1;19;
8;conditional;foo_1 < foo_2;20;
9;conditional;(fun_1(foo_2) != 'a') || (fun_2(foo_4) != 'b');21;
10;statement;return false;22;
11;statement;i++;24;
12;statement;j--;25;
13;statement;return true;27;
Edges
0;2;
1;3;
2;4;6;
3;5;
4;
5;7;
6;8;
7;13;9;
8;10;11;
9;5;
10;12;
11;8;
12;5;
